debian/cocaine-v12-runtime.init

```diff
@@ -14,7 +14,7 @@
 # Author: Andrey Sibiryov <kobolog@yandex-team.ru>
 
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
-NAME=cocaine-runtime
+NAME=cocaine-v12-runtime
```

Comment:
To have the same name which the package have. Otherwise, the init script would not find ```defaults``` and binary.